### PR TITLE
Update README.md for MAC setup

### PR DESCRIPTION
**Pull request by Tony911029**
Add some description for mac setup
 - Security issues
 - Rename folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,8 +107,22 @@ brew info open-cd
 
 4. install [the compiler toolchain](https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-mac.tar.bz2?revision=d0d318de-b746-489f-98b0-9d89648ce910&hash=DB1DA90A2BC0E5A0D3FA92D4E7D2E9A6F4A2118D)
 5. Unzip the package you just installed and put it in the directory /Applications/ARM (you may need to create this directory)
-6. in the file /etc/paths add the directory /Applications/ARM/arm-none-eabi-gcc/bin. ([see the original instructions](https://gist.github.com/disposedtrolley/06d37e1db82b80ccf8c5d801eaa29373))
-7. Try to build the firmware, run `make all`
+6. Rename the folder name in the directory /Applications/ARM to arm-non-eabi-gcc (old name should be something like this: gcc-arm-none-eabi-9-2020-q2-update-mac)
```
Comment (zzakiullah) [inline README.md:110]: If the following steps only applies to M1 Mac devices, can you make a note of that before these steps?

File: README.md [hunk 2]
```diff
@@ -107,8 +107,22 @@ brew info open-cd
...
+7. in the file /etc/paths add the directory /Applications/ARM/arm-none-eabi-gcc/bin. ([see the original instructions](https://gist.github.com/disposedtrolley/06d37e1db82b80ccf8c5d801eaa29373))
```
Comment (zzakiullah) [inline README.md:111]: Surround `/etc/paths` and `/Applications/ARM/arm-none-eabi-gcc/bin` with \` just so it's more clear to new members exactly what the directories/files encompass

File: README.md [hunk 3]
```diff
@@ -107,8 +107,22 @@ brew info open-cd
...
+    - In the terminal, enter: `sudo nano /etc/paths` which will open the paths file
+    - Add `/Applications/ARM/arm-none-eabi-gcc/bin` to the file
+    - Control + X ->  Enter Y -> Hit Enter to save the file
+    - Quit the terminal instance
+8. Try to build the firmware, run `make all`
+9. There are some security issues with ARM Mac when you run `make all`
+    - It should give you something about "file name" with an unverified developer message
+    - Please do **NOT** click delete the file -> click cancel
+    - Copy the file name such as "ar" or "as" and search the executable with the same name in the folder arm-none-eabi-gcc you just renamed
```
Comment (zzakiullah) [inline README.md:120]: If this involves a bash command, maybe say what the command is or an example of how you'd use it?

File: README.md (hunk 2 above)
Comment (zzakiullah) [inline README.md:111]: Also if this involves running `sudo vim` or `sudo nano` maybe give an example of how you'd use it to add the directory to the /etc/paths file

File: README.md (hunk 3 above)
Comment (Tony911029) [reply README.md]: This step does not involve a bash command. Just have to open the file in the folder manually to bypass the security.

**Review by addison-ch — APPROVED** [review]

**Review by zzakiullah — APPROVED** [review]

**Pull request merged**